src/main/java/com/dev/cinema/exeptions/AuthenticationException.java

```diff
@@ -0,0 +1,7 @@
+package com.dev.cinema.exeptions;
+
+public class AuthenticationException extends RuntimeException {
```

Comment:
From the checklist:
Custom AuthenticationException shouldn’t be RuntimeException

Reply:
done